### ISSUE TITLE
Upgrade OpsMan Pipeline Fails with Windows Stemcells on vSphere

### ISSUE THREAD

**Issue by cdelashmutt-pivotal** [pivotal-cf/pcf-pipelines#161]
When trying to use the vSphere Upgrade OpsMan pipeline for a deployment that uses BOSH deployed Windows Diego Cells, the "download-stemcells" task in the "upgrade-opsmgr" job fails with the error: `Could not find the required stemcell version XXXX.X. This version might not be published on PivNet yet, try again later.` where `XXXX.X` is the version number for the Windows Stemcell you have deployed.

PivNet can't host Windows Stemcells for vSphere due to constraints on redistribution of Windows.  This doesn't happen for Public IaaS since "light" stemcells are provided by PivNet for Windows.

It would be great if I had a way to supply the Windows stemcells that I have to build into the Upgrade OpsMan pipeline (and any other pipelines that need Windows stemcells on vSphere) so that the pipeline could complete without failures.

**Comment by christianang** [comment]
We talked over slack, but to get some of our discussion down in the issue.

I created a operations file: [operations/add-download-custom-stemcell-from-s3](https://github.com/pivotal-cf/pcf-pipelines/blob/master/operations/add-download-custom-stemcell-from-s3.yml) that can patched into the vSphere upgrade-ops-manager pipeline. This ops file will replace the `download-pivnet-stemcells` task with the `download-custom-stemcells-from-s3` task.

This does mean you will have to upload all necessary stemcells to your s3 bucket, and not just the windows stemcell.
